packages/grafana-data/src/types/displayValue.ts

```diff
@@ -20,7 +22,10 @@ export interface DisplayValue extends FormattedValue {
    */
   icon?: string;
   title?: string;
-
+  /**
+   *  FieldType for use when adding custom prefixes/suffixes to specific types of values
+   */
+  valueType?: FieldType;
```

Comment:
for sure we do not want to add type here... if anything (and i don't think we need anything) lets attach the source field, then we will have it there once and for all :)

Reply:
Yep; we don't need this anyway.